### ISSUE TITLE
natural controls and scrolling

### ISSUE THREAD

**Issue by smalls** [GoogleWebComponents/model-viewer#75]
Let's look at making the interaction between controls and scrolling better, and more natural.

The issue now is that the model element, with controls enabled (after #56 is fixed), captures all events, including those that would scroll the page. On mobile where the element takes up a significant portion of screen real estate we may end up preventing any scrolling past a certain point.

We discussed some solutions that we think have potential, including (I may be missing some):
- By default, having the model only move vertically a small amount; after that reaches the maximum, go back to scrolling the page
- Pass initial vertical movements up to the page; the user would have to interact with the model by rotating it horizontally before they could rotate it vertically

Other options:
- Using 2 fingers to move the model, a single finger scrolls the page
- A button to toggle model controls

**Comment by smalls** [comment]
Let's get a prototype together, but we may want to iterate after that. @cdata may be able to split this out into some smaller issues.

**Comment by cdata** [comment]
Some concrete improvements being made in the first pass:

 - Velocity-based camera movement
    - `OrbitControls` uses a "damping"  approach, which usually results in the camera overshooting its target / limits and only eases when slowing down.
 - Local-only (e.g., per-element) handling of keyboard input
    - `OrbitControls` uses global listeners for keyboard input
 - Keyboard arrows control orbital position by default
    - `OrbitControls` uses arrow keys to control panning
 - Allow unhandled events to escape control handlers
    - This will enable us to e.g., continue scrolling once the camera has reached the max polar angle
    - `OrbitControls` consumes all events that it handles, even if it effectively does nothing with them
